src/tests/Interop/PInvoke/Vector2_3_4/Vector2_3_4.cs

```diff
@@ -5,29 +5,15 @@
 using System.Numerics;
 using System.Runtime.InteropServices;
 using TestLibrary;
+using FactAttribute = Xunit.FactAttribute;
```

Comment:
Is there anything that prevents us from just saying <code>using Xunit</code>? The explicit selective usings seem somewhat wordy and ugly especially for tests that need additional attributes like <code>ConditionalFact</code>. If the full using of the Xunit namespace causes some aliasing conflict with the <code>TestLibrary</code> or some such, we should probably look into fixing it if possible.